README.md

```diff
@@ -359,6 +371,7 @@ The following descriptions provide additional elaboration on a few common parame
 | `--print` | Use `mockery --print` to have the resulting code printed out instead of written to disk. |
 | `--exported` | Use `mockery --exported` to generate public mocks for private interfaces. |
 | `--with-expecter` | Use `mockery --with-expecter` to generate `EXPECT()` methods for your mocks. This is the preferred way to setup your mocks. |
+| `--log-level` | Use `--log-level` to generate useful logs. Possible values are `trace`, `debug`, `info`, `warn`, `error`, `fatal`, `panic` |
```

Comment:
This is pretty self-explanatory from the CLI, probably unnecessary here in my view.